lib/src/test/java/org/team100/lib/targeting/InterceptTest.java

```diff
@@ -27,7 +28,7 @@ void testBothStationary() {
         assertEquals(0, azimuth.get().getRadians(), DELTA);
     }
 
-    // @Test
+    @Test
```

Comment:
you'll want to enable the test above this one, note the "@Test" is commented out.